biz.aQute.bnd.annotation/bnd.bnd

```diff
@@ -8,4 +8,5 @@ Bundle-Description:     bnd Annotations Library
 	org.osgi.namespace.extender,\
 	org.osgi.namespace.service,\
 	org.osgi.resource;version=latest,\
-	org.osgi.service.serviceloader;version=latest
+	org.osgi.service.serviceloader;version=latest,\
+	osgi.core
```

Comment:
I think you just need org.osgi.resource and not osgi.core.

Reply:
done